scripts/validate-engine-versions

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# This script will check the following versions provided in package.json,
+# based on current versions of node and npm
+# {
+#   ...
+#   "engines": {
+#     "node": "4.4.x",
+#     "npm": "3.9.x"
+#   }
+#   ...
+# }
+#
+PACKAGE_NODE_VERSION=$(node -p -e "require('./package.json').engines.node")
+NODE_VERSION=$(node -v)
+if [[ "v${PACKAGE_NODE_VERSION%.*}" == "${NODE_VERSION%.*}" ]]; then
+  echo "Node version is valid ($NODE_VERSION)"
+else
+  echo "ERROR: Node version v${PACKAGE_NODE_VERSION%.*}.x does not match required version ${NODE_VERSION%.*}.x"
```

Comment:
These two version variables need to be flipped: `echo "ERROR: Node version v${NODE_VERSION%.*}.x does not match required version ${PACKAGE_NODE_VERSION%.*}.x"`